backend/LexBoxApi/Services/ProjectService.cs

```diff
@@ -59,4 +60,15 @@ await _dbContext.Projects.Where(p => p.Code == projectCode)
             );
         return lastCommitFromHg;
     }
+
+    public async Task<ProjectMigrationStatus> GetProjectMigrationStatus(string projectCode)
+    {
+        //todo remove this when we have the migration info in the db
```

Comment:
Isn't it in the db now?

Reply:
haha yes it is, I meant to come back for that